frontend/friendsAPIMock.js

```diff
@@ -0,0 +1,76 @@
+// mock functions for friendsAPi.js
+
+// returns res.status 200
+// function not used
+const createFriendshipTestMock = async () => {
+    return 200;
```

Comment:
return Promise.resolve(200) 
need to return a promise.resolve for every function

Reply:
also remove this test function